inference/infer_assertions.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin bash
```

Comment:
Oh my, I didn't even notice the bash files in the previous PR! ...my bad. Anyway – no, bash files for CI should not be committed to the main repo, that's not a good practice. So, sorry, but I don't see any way you can avoid modifying the ci-worker. Please:
- Move the bash files to the ci-worker repo (create a new directory for inference, perhaps)
- Modify the ci-worker's Dockerfile to add a step where you download ROBOT's JAR via wget
- Update this pull request accordingly

...and setting up Java can stay as a CI step here, it's acceptable for now.